### PR TITLE
Add file header for npm.el

### PR DESCRIPTION
**Pull request by zakame**
Fixes azer/npm.el#5

### CODE REVIEW

File: npm.el [hunk 1]
```diff
@@ -1,3 +1,27 @@
...
+
+;; Author: Azer Koçulu
+;; Keywords: languages, javascript, convenience
+;; Package-Requires: ((emacs "24.1"))
+
+;; This file is NOT part of GNU Emacs.
+
+;;; Commentary:
+
+;; npm.el makes it convenient to create new NodeJS projects within Emacs.
```
Comment (scottaj) [inline npm.el]: I would change this to 

> npm.el makes it convenient to create and manage npm projects within Emacs

**Comment by scottaj** [comment]
This is great. It's been on my todo list for a little bit now to get this into MELPA, I just haven't found the time lately.

This is great start towards that goal, from my past experience getting a package on MELPA, I think some changes to the code will be needed to (they have pretty strict standards).

Some things I can think of that will be needed:
1. changing all the free variables (e.g. the setq's at the beginning of the file) to be defcustoms or defvars
1. Adding autoloads to all the public functions

Those changes are outside the scope of this PR, but are you interested in doing any of that work?

**Comment by zakame** [comment]
@scottaj sure, I'm already thinking about doing the changes to the free variables and autoloads.  I'll update this PR :muscle:

**Comment by scottaj** [comment]
@zakame Are you still working on updates to this? If so are you planning on updating this PR or do you want me to merge just this and put more changes in another PR?

**Comment by zakame** [comment]
@scottaj yeah, sorry I haven't updated this PR beyond touching the commentary.  I think I can make another PR for changing variables and autoloads within this week.

**Comment by scottaj** [comment]
Sounds good, I'm going to merge this then.

**Pull request merged**